Clarified `.github/workflows` name importance in `quickstart.md` (#31837)
github/docs: content/actions/quickstart.md

## Creating your first workflow

> 1. Create a `.github/workflows` directory in  your repository on {% data variables.product.prodname_dotcom %} if this directory does not already exist. The `.github/workflows` directory has special significance to {% data variables.product.prodname_dotcom %} - any file placed in this directory is automatically discovered and registered as a workflow in {% data variables.product.prodname_actions %}.

Reviewer: minor change to make the wording more direct
```suggestion
1. Create a `.github/workflows` directory in  your repository on {% data variables.product.prodname_dotcom %} if this directory does not already exist. The directory must have this exact name in order for {% data variables.product.prodname_dotcom %} to discover any {% data variables.product.prodname_actions %} workflows that it contains.
```

> 1. In the `.github/workflows` directory, create a file named `github-actions-demo.yml`. For more information, see "[AUTOTITLE](/repositories/working-with-files/managing-files/creating-new-files)." The name of this file does not matter as long as it ends in `.yml` or `.yaml`.

Reviewer: Since later steps refer to this name, we can swap this around:
```suggestion
1. In the `.github/workflows` directory, create a file with the `.yml` or `.yaml` extension. This tutorial will use `github-actions-demo.yml` as the file name. For more information, see "[AUTOTITLE](/repositories/working-with-files/managing-files/creating-new-files)."
```
